modules/calib3d/include/opencv2/calib3d/calib3d.hpp

```diff
@@ -490,6 +490,21 @@ CV_EXPORTS_W void solvePnPRansac( InputArray objectPoints,
                                   OutputArray inliers = noArray(),
                                   int flags = ITERATIVE);
 
+//! computes the camera pose from a few 3D points and the corresponding projections. The outliers are possible.
+CV_EXPORTS_W void solvePnPRansac( InputArray objectPoints,
```

Comment:
don't introduce another function, just add an optional parameter to the original one.